engine/src/main/java/com/datatorrent/stram/engine/StreamingContainer.java

```diff
@@ -1290,6 +1290,7 @@ private void deployInputStreams(List<OperatorDeployInfo> operatorList, HashMap<S
    */
   private void setupOiOGroups(Map<Integer, Integer> oioNodes)
   {
+    oioGroups.clear(); //Reset prior info if any before redeployment
```

Comment:
should not clean up be handled in undeploy()? What if AM decides to deploy another operator/partition to a container that has few operators with thread local stream deployed? Will not that lead to the existing OiO groups being lost?